### PR TITLE
Five digit commands

### PR DESCRIPTION
**Pull request by QualityError**
Accepts 5 digit commands and handles them correctly

also updates the py test to use 5 digit commands

### CODE REVIEW

File: memory.h [hunk 1]
```diff
@@ -17,8 +17,8 @@ class Memory {
         void print_memory();
         void dumpMemory();
     
-    private://should not be used out =side of memory.cpp
-        int memory_array[capacity];
+    private://should not be used outside of memory.cpp
+        int memory_array[1000];//need a constructor method if you want to use capacity to define
```
Comment (jansenfuller) [inline memory.h:21]: Can you elaborate on this more

Comment (QualityError) [reply memory.h:21]: Got errors and could not compile using a variable in the object specification. Using a hard value worked. Might be able to be fixed with c++'s version of "this.capacity"

Comment (QualityError) [reply memory.h:21]: changed it back for this PR

**Review by DCprivate — APPROVED** [review]
Go over with team so we all know how it works

**Pull request merged**